lib/tasks/mastodon.rake

```diff
@@ -45,6 +45,7 @@ namespace :mastodon do
         env[key] = SecureRandom.alphanumeric(32)
       end
 
+      # Uses Webpush class, Webpush::Legacy does the same thing.
```

Comment:
I think this can be removed, `Webpush::Legacy` (as in, in the gem maintained by Mastodon) does not cover this at all.